recipe/meta.yaml

```
@@ -1,50 +1,44 @@
 {% set name = "hkl" %}
-{% set version = "5.0.0.2173" %}
-{% set sha256 = "5f3991d2af4ff01d6f6e29d0176af9bc1b54e6a8598046a050ad804fedd860f2" %}
+{% set gitorg = "bluesky" %}
+{% set version = "v5.0.0.3001" %}
+{% set license = "COPYING" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/picca/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: {{ sha256 }}
-  patches:
-    - no-docs.patch
+  git_rev: main
+  git_url: https://github.com/{{ gitorg }}/libhkl.git
 
 build:
-  number: 7
-  skip: True  # [py<36 or win or osx]
+  number: 0
+  skip: True  # [py<38 or py>11 or win or osx]
 
 requirements:
-  build:
-    - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - autoconf
-    - automake
-    - libtool
-    - m4
-    - make
-    - pkg-config
   host:
-    - glib
-    - gobject-introspection
-    - gsl
-    - libcblas
-    - libiconv
-    - pcre
+    - gobject-introspection =1.78.1
+    - gsl =2.7
     - pygobject
     - python
   run:
     - python
-    - gsl
+    - gobject-introspection =1.78.1
+    - gsl =2.7
     - numpy
     - pygobject
+    # - conda-verify  ?
 
 test:
   commands:
+    - ls -lAFgh ${PREFIX}/lib/libhkl.*
     - test -f ${PREFIX}/lib/libhkl.so
```

We'd like to also see how it resolves the dependency libraries:
```suggestion
    - test -f ${PREFIX}/lib/libhkl.so
    - ldd ${PREFIX}/lib/libhkl.so
```